Dockerfile

```
@@ -66,6 +66,11 @@ USER 65532:65532
 
 ENTRYPOINT ["/vault-secrets-operator"]
 
+# ubi build image
+# -----------------------------------
+FROM registry.access.redhat.com/ubi9/ubi-minimal:latest as build-ubi
```

We may want to explicitly version the ubi-minimal image as well:
```suggestion
FROM registry.access.redhat.com/ubi9/ubi-minimal:9.2-750.1697625013 as build-ubi
```

---

Seems reasonable - I wonder if we should always be pulling latest for both minimal and micro, that approach would be in-line with our distroless builds.

---

`latest` would certainly be convenient, though it makes it a little harder to tell which image was used for a release (when tracking down CVEs, etc.) `distroless` seems to be kind of an outlier since it doesn't have tagged versions.

---

Agreed. I prefer an explicit version over one derived at build time.